### DISCUSSION TITLE
look at my code and help me

### DISCUSSION THREAD

**Discussion by notforgetwyn** [community/community#29842] [New to GitHub]
_**this is my first time to writing code，I want to gain the data and keep it to a file（wn.html）**_
`import requests
ad1='https://m.bcoderss.com/'
ad2={'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/103.0.0.0 Safari/537.36'}
ad3=requests.get(ad1,headers=ad2,)
name=ad3.text
ad4=str(name)
with open('wn.html',mode='w') as ad5:
    ad5.write(ad4)`
**this is the result**
`Traceback (most recent call last):
  File "C:\Users\Administrator\PycharmProjects\pythonProject3\main.py", line 8, in <module>
    ad5.write(ad4)
UnicodeEncodeError: 'gbk' codec can't encode character '\ufeff' in position 0: illegal multibyte sequence
`

**Comment by airtower-luna** [comment]
Could you put your code in a [code block](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks), so lines and indentation are preserved? It's hard to impossible to read otherwise.
